software/proposals/volunterring_system.md

```diff
@@ -1,16 +1,19 @@
 | | |
 |-|-|
 | Proposal Stage | Draft     |
-| Tags           |      |
+| Tags           | volunteer, impact     |
 | Created Date   |      |
 | Lead           | Charles He     |
-| Slack Channel  |      |
+| Slack Channel  | project-volunteer_app     |
 
 ## 📃 Idea in brief: Question to be resolved and why
 
 ### 💥 Problem
 
-There are capable people who are recovered and immune to the virus and were trained in anti-septic techniques.These people could volunteer at hospitals, performing menial but critical roles in assisting nurses.
+A general volunteer systes that helps people find opportunities to help others. Volunteers will be able to flag if they are comfortable working with sick people, and organizations can on-board volunteers quickly and easily.
+
+For example, there are capable people who are recovered and immune to the virus and were trained in anti-septic techniques.These people could volunteer at hospitals, performing menial but critical roles in assisting nurses.
```

Comment:
in a Slack discussion it was mentioned that might not be true that recovered people are immune. Could you add a small note saying `pending to verify` or something similar?

Reply:
Yeah, as I understand it, this virus is mutating so fast that there are multiple strains that people could get re-infected with. I think doctors have identified 4 strains. There still may be some young/strong people willing to help sicker people regardless of their risk.